README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# Biography
+
+My name is **Kenna Rearick**. I am currently attending the University of Chicago, where I am a **graduate student** in the **Social Service Administration** Advanced Standing Program. I am also a research assistant intern at the University of Chicago **Urban Labs**, where I assist with the Supportive Release Center research study. 
+
+Prior to attending UChicago, I graduated from Youngstown State University with my **Bachelor's in Social Work**. My prior experience includes frontline work with children and families who have experienced homelessness, domestic violence, and other forms of trauma. I am also a proud [AmeriCorps NCCC](https://www.nationalservice.gov/programs/americorps/americorps-programs/americorps-nccc) alum. During my time in AmeriCorps, I traveled through more than 20 states and I lived in the following locations:
+
+* Baltimore, MD
+* Washington, DC
+* New York, NY
+* Seattle, WA
+* Stayton, OR
+
+I currently live in Hyde Park with my cat, Venus. ![Venus](https://i.imgur.com/9XmLxVp.jpg)
```

Comment:
good use of bullet point and I enjoy your picture. Your assignment satisifies all the requirements. Good job! The only suggestion I would like to say: trying to segment your body paragraphy so as to make your statement in an organized way.